tests/e2e/autoscale/03-assert.yaml

```diff
@@ -6,3 +6,11 @@ metadata:
 status:
   scale:
     replicas: 2
+---
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: simplest-set-utilization
```

Comment:
Before your changes, the test was done against the `simplest` `OpenTelemetryCollector` instance. That one is just to check the old behavior still works. But it would make more sense to do the test against the `simplest-set-utilization` `OpenTelemetryCollector` instance (and will simplify your test).

Reply:
Hmm is your recommendation to remove `simplest` collector entirely? I think I'd like to keep it around for now because it also confirms HPA is created with default `targetCPUUtilization` (i.e. scale up  and scale down happens without having to explicitly set the cpu utilization because the default is used).

Reply:
Happy to continue on e2e tests improvements on my followup PR though, so let me know what you think!

Reply:
> Hmm is your recommendation to remove simplest collector entirely? I think I'd like to keep it around for now because it also confirms HPA is created with default targetCPUUtilization (i.e. scale up and scale down happens without having to explicitly set the cpu utilization because the default is used).

Not removing it completely but not checking if it scales. Just checking if the new approach is the one that scales up and down and can be updated (as you did).

Reply:
Ah I see, sounds good to me! I now removed the tracegen and scaleup/scaledown assertions from the test for the `simplest` collector. Thanks for the suggestions @iblancasa !